services/scabbard/cli/man/scabbard-ns-create.1.md

```diff
@@ -0,0 +1,94 @@
+% SCABBARD-NS-CREATE(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**scabbard-ns-create** — Creates a Sabre namespace.
+
+SYNOPSIS
+========
+
+**scabbard ns create** \[**FLAGS**\] \[**OPTIONS**\] NAMESPACE
+
+DESCRIPTION
+===========
+This command allows users to create a new Sabre namespace in state for the
+targeted scabbard service.
+
```

Comment:
Should the description mention using `scabbard perm` to set the namespace permissions?  If so, add `scabbard-perm(1)` to the SEE ALSO section.

Reply:
Added